Library/Homebrew/cmd/test-bot.rb

```diff
@@ -572,6 +582,14 @@ def formulae
       changed_formulae + unchanged_formulae
     end
 
+    def head_only?(formula)
+      formula.head && formula.devel.nil? && formula.stable.nil? && tap = "homebrew/homebrew-head-only"
```

Comment:
Problem goes away if we switch to ` formula.tap ` so happy with that if you are.